vendor/bat-native-ledger/src/bat/ledger/internal/uphold/uphold_wallet.cc

```diff
@@ -297,15 +297,18 @@ void UpholdWallet::OnLinkWallet(const type::Result result,
     return callback(type::Result::LEDGER_ERROR);
   }
 
-  LogWalletStatusChange(ledger_, from, to);
+  WalletStatusChange(ledger_, from, to);
 
   ledger_->database()->SaveEventLog(
       log::kWalletVerified,
       constant::kWalletUphold + std::string("/") + id.substr(0, 5));
 
-  ledger_->ledger_client()->ShowNotification(
-      ledger::notifications::kWalletNewVerified, {"Uphold"},
-      [](type::Result) {});
+  extern bool show_wallet_new_verified;
+  if (show_wallet_new_verified) {
```

Comment:
I think it'd be better to expose this as a `ShouldShowNewlyVerifiedWallet` function than to allow other modules to access this directly.